docs/set_delegate.md

```diff
@@ -41,7 +54,7 @@ A call to the KT1's smart contract's `do` method must be made to delegate to a K
 >
 > For the time being, we regard KT1 manager accounts as a regular smart contract. It is possible to have a smart contract that is not following the manager.tz conventions, and that also delegates to a baker. The correct lambda to pass to a contract to delegate is application/wallet specific. Therefore Taquito does not make any assumption on the KT1.
```

Comment:
agreement:  "we regard KT1 manager accounts as a regular smart contract" 

either "we regard a KT1 manager account as a regular smart contract"
or
"we regard KT1 manager accounts as regular smart contracts"